format_checker/gr_checker.py

```diff
@@ -0,0 +1,186 @@
+"""Implements rule checks for the gr-data.csv file."""
```

Comment:
I see more files now. Instead of having a copy of PR checker in GR checker, let's (1) make PR checker more general to take an input file name, and (2) run PR checker with both `pr-data.csv` and `gr-data.csv`.

Reply:
https://github.com/TestingResearchIllinois/idoft/pull/770#issuecomment-1704336924